### PR TITLE
[improvement](docs): add doc of fe dev in vscode

### PR DESCRIPTION
**Pull request by jackwener**
# Proposed changes

None

## Problem Summary:

Add doc of fe dev with vscode.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments
None

### CODE REVIEW

**Comment by luzhijing** [comment]
LGTM

File: docs/zh-CN/developer-guide/fe-vscode-dev.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## 下载代码编译
+
+1. 从 https://github.com/apache/incubator-doris.git 下载源码到本地
+
+2. 使用 VSCode 打开代码 `/fe` 目录
+
+## 配置 VSCode
+
+在 `.vscode` 目录下创建 `settings.json` 文件, 分别配置
```
Comment (EmmyMiao87) [inline docs/zh-CN/developer-guide/fe-vscode-dev.md:46]: vscode 里不需要配置 远程 ip 端口嘛？

Comment (jackwener) [reply docs/zh-CN/developer-guide/fe-vscode-dev.md:46]: > vscode 里不需要配置 远程 ip 端口嘛？

不需要额外配置，在启动参数了加了就够了。vscode有默认配置的。

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**